relate/bin/relate.py

```diff
@@ -305,6 +328,12 @@ def main() -> None:
     parser_expand_yaml.add_argument("YAML_FILE")
     parser_expand_yaml.set_defaults(func=expand_yaml_ui)
 
+    parser_lint_yaml = subp.add_parser("lint_yaml")
+    parser_lint_yaml.add_argument("--repo-root", default=os.getcwd())
+    parser_lint_yaml.add_argument("--config-file", default="./.yamllint")
+    parser_lint_yaml.add_argument("DIRECTORY")
```

Comment:
I'd keep this lower case. (You can set `metavar=` to have it shown upper case in `--help`.)